lib/std/os/linux.zig

```diff
@@ -745,33 +745,33 @@ pub fn setregid(rgid: gid_t, egid: gid_t) usize {
 
 pub fn getuid() uid_t {
     if (@hasField(SYS, "getuid32")) {
-        return @as(uid_t, syscall0(.getuid32));
+        return @truncate(uid_t, syscall0(.getuid32));
```

Comment:
Shouldn't this be an `@intCast`? We don't actually expect values larger than `uid_t`.

Reply:
Sounds reasonable, replaced `@truncate` with `@intCast`.